src/rotator_library/utils/resilient_io.py

```
@@ -677,3 +678,66 @@ def safe_mkdir(path: Union[str, Path], logger: logging.Logger) -> bool:
         return False
+
+
+# =============================================================================
+# ASYNC RESILIENT STATE WRITER
+# =============================================================================
+
+
+class AsyncResilientStateWriter:
+    """Надежный асинхронный записыватель state."""
+
+    def __init__(self, file_path: Path, flush_interval: float = 5.0):
```

Ruff (0.15.4)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)